docs/content/management-bundles.md

```diff
@@ -569,11 +571,25 @@ The `"value"` field defines the value to be added or replaced. Only required for
 
 #### Limitations
 
-* _Delta_ bundles only support updates to data
+* _Delta_ bundles only support updates to data. Policies cannot be updated using _delta_ bundles.
+* _Delta_ bundles do not support bundle signing.
+* Unlike _snapshot_ bundles, activated _delta_ bundles are not persisted to disk when the `bundles[_].persist` field is `true`.
+* An empty list of operations in a _delta_ bundle `patch.json` will remove all the data from OPA's in-memory store. I.e., the following are equivalent:
 
-* _Delta_ bundles do not support bundle signing
+  ```json
+  {
+    "data": []
+  }
+  ```
 
-* Unlike _snapshot_ bundles, activated _delta_ bundles are not persisted to disk when the `bundles[_].persist` field is `true`
+  ```json
+  {
+    "data": [
+      { "op": "replace", "path": "/", "value": {} }
+    ]
+  }
+  ```
+  If there are no operations to apply to the data, the bundle server should return the same [`Etag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) value as the last update. OPA will also send the last `Etag` value in the [`If-None-Match`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match) Header.
```

Comment:
Nit: Instead of adding this here, wdyt about adding it as a warning or info under the `Delta Bundle File Format` section?

Reply:
I think that would make it more explicit and highlight current behavior that one or more patch ops should be provided.

Reply:
Thanks, I think that makes more sense. See 2536f44b48f73ace383660522ff84c3e7b438581